common/changes/office-ui-fabric-react/anihan-commandBar-menu-shadow_2017-07-10-23-32.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add variant of Callout with no shadow on one edge",
+      "type": "minor"
+    }
+  ],
+  "packageName": "office-ui-fabric-react",
+  "email": "anihan@microsoft.com"
+}
```

Comment:
If you do `rush change -b upstream` you won't get all of these extraneous change files.